backend/src/formInstance/formInstance.service.ts

```diff
@@ -1,15 +1,25 @@
 import { Injectable } from "@nestjs/common";
 import { InjectRepository } from "@nestjs/typeorm";
 import { FormInstance } from "../models/formInstance.entity";
-import { DataSource, Repository } from "typeorm";
+import { Repository } from "typeorm";
 import { CreateFormInstanceDto, FormInstanceDto } from "./formInstance.dto";
 
 @Injectable()
 export class FormInstanceService {
     constructor(
         @InjectRepository(FormInstance)
-        private formInstanceRepository: Repository<FormInstance>) {}
+        private formInstanceRepository: Repository<FormInstance>
+    ) {}
 
     public async createFormInstance(createFormInstanceDto: CreateFormInstanceDto) {
+        const formInstanceDto: FormInstanceDto = {
+            formId: createFormInstanceDto.formId,
+            completed: false,
+            signatureRequestHead: null,
+            initiatorId: createFormInstanceDto.employeeId
```

Comment:
So currently we're passing in the employeeId, while initiatorId should be a positionId. We should probably change the DTO to be positionId instead of employeeId